src/components/Badge/types.ts

```diff
@@ -4,31 +4,28 @@ import { IconProps } from '@foundation/Icon/IconProps';
 import { PropsWithChildren, ReactElement } from 'react';
 import { Color } from '../../types';
 
+export type BadgeSize = 's' | 'm';
```

Comment:
Thanks!! Looks good to me 🎉  I would just update that little type to be in full letters. Even if it's quite understandable, just for consistency as you already updated the RGBA. 😄

Reply:
Sure, fixed :) Will introduce a breaking change though

Reply:
Thanks! No problem so did Jonas by removing the self-alignment on that same component. 😄